packages/webdriverjs/example.ts

```diff
@@ -0,0 +1,14 @@
+import AxeBuilder from './src';
+import * as WebDriver from 'selenium-webdriver';
+
+const driver = new WebDriver.Builder().forBrowser('chrome').build();
+
+driver.get('https://html5-sandbox.glitch.me/').then(() => {
+  const axe = new AxeBuilder({ driver, builderOptions: { noSandbox: true } });
+  axe.analyze((err, results) => {
```

Comment:
Thoughts on calling this file `example-with-callback.js` and creating an `example.js` which uses `async/await` to get results? At this point, I think most developers are familiar enough with `async/await` for the syntax to be used in the "main example".